### DISCUSSION TITLE
First project

### DISCUSSION THREAD

**Discussion by rmoortgat** [OpenRadioss/OpenRadioss#2269] [Q&A] [answered]
Good morning,

I have some experience with FEA, salome-meca and DIANA (long time ago) but I have absolutely no knowledge about  Radioss.
I the meantime I got OpenRadioss up and running and I did some of the examples.
Now I would like to do a simulation by myself of an elastic mass falling on an elastic surface. Say a ball on a steeldeck roof.
I can mesh with gmsh an postprocess with Paraview, but I have no idea where to start with the other decks, i.e. materials, BC, ... Is there a tutorial available that can get me started without reading the thousands of pages of the user manual and reference? A simple step by step tutorial would do.

Roger

**Comment by MarianBulla** [comment, marked as answer]
Good morning Roger,
thank you very much for your interest in OpenRadioss!
Yes sure, you can find many small models here: 

https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/8749071/Example+Models

Showing many features.

Especially the Bumper-Beam model contains many of the features, you need. 
Just open the model file with your favorite text editor and have a look for the keywords starting with "/".
In the documentation section, you will the deeper explanations for all the keawords.

Please review also the components section, here at OpenRadioss GitHub repsoitory, for very simple material models:

https://github.com/OpenRadioss/ModelExchange/tree/main/Components

(by the way, this repository is open to everyone, to provide such simple models for demonstrating some features, which can be of interest to others. So, feel free to upload your models, to help others getting faster on speed with OpenRadioss)

Hope it helps you ?

Thanks and best regards,
Marian
